pkg/agent/openflow/pipeline.go

```diff
@@ -332,6 +353,7 @@ type OFEntryOperations interface {
 	Modify(flow binding.Flow) error
 	Delete(flow binding.Flow) error
 	AddAll(flows []binding.Flow) error
+	ModifyAll(flows []binding.Flow) error
```

Comment:
why is the refactor? what will the method be used for?

Reply:
The refactor helps reuse the code in `Add()`, `Delete()`, etc.
`ModifyAll()` now is used by `Modify()` and can be used for other cases that several flows are modified.